server_tests/test/clientTests.js

```diff
@@ -1486,11 +1379,11 @@ describe("RUST API TEST SUITE", function() {
                     }
                 }
             )
-            .then(Comparison.catchExceptions(done, function(res) {
+            .then(function onSuccess (res) {
                 Comparison.oscSetOptionsOutput(res);
                 return testClient.bublTimelapse(sessionId);
-            }))
-            .then(Comparison.catchExceptions(done, function(res) {
+            }, shouldNotGetHere)
+            .then(function onSuccess (res) {
```

Comment:
`done()` is still used in here. Use `assert.fail()` instead or `shouldNotGetHere()` etc.